controllers/osbuild_controller.go

```diff
@@ -107,60 +100,45 @@ func (r *OSBuildReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 		return ctrl.Result{}, nil
 	}
 
-	if osBuild.Spec.Details != nil {
-		// build edge-container image
-		return r.buildEdgeContainerImage(ctx, logger, osBuild)
-	} else if osBuild.Spec.EdgeInstallerDetails != nil {
-		return r.buildEdgeInstallerImage(ctx, logger, osBuild)
+	if osBuild.Status.ComposeId == EmptyComposeID {
+		// if the image wasn't created yet - schedule a new build
+		logger.Info("create a new image")
+		// TODO: if target image type is edge-installer call postComposeEdgeInstallerImage
```

Comment:
Just want to make sure. This doesn't create a loop of builds if I try to build an installer, right? (causing the OSBuildConfig controller to keep scheduling it)

Reply:
Right - this is a todo comment for checking if the post compose request fo edge-installer is different and might be done in a different method

Reply:
Please mark it as done if it's fine by you